src/components/registration/steps.scss

```diff
@@ -212,5 +212,57 @@
                 }
             }
         }
+
+        &.demographics-step {
+            .form-group {
+                &.has-error {
+                    .gender-input {
+                        transform: translateY(-3rem);
+                    }
+                }
+            }
+        }
+
+        &.organization-step {
+            .checkbox-group {
+                .validation-message {
+                    $arrow-border-width: 1rem;
+                    display: block;
+                    position: absolute;
+                    top: 0;
+                    left: 0;
+                    transform: translate(16rem, -16rem);
+                    margin-left: $arrow-border-width;
+                    border: 1px solid $active-gray;
+                    border-radius: 5px;
+                    background-color: $ui-orange;
+                    padding: 1rem;
+                    max-width: 18.75rem;
+                    min-height: 1rem;
+                    max-height: 3rem;
+                    overflow: visible;
+                    color: $type-white;
+
+                    &:before {
+                        display: block;
+                        position: absolute;
+                        top: 1rem;
+                        left: -$arrow-border-width / 2;
+
+                        transform: rotate(45deg);
+
+                        border-bottom: 1px solid $active-gray;
+                        border-left: 1px solid $active-gray;
+                        border-radius: 5px;
+
+                        background-color: $ui-orange;
+                        width: $arrow-border-width;
+                        height: $arrow-border-width;
+
+                        content: "";
```

Comment:
Ideally if it's a general rule, but if it's not then it's better to only override the ones you need to override so you don't have to reverse the rules immediately after.  If the arrow is ok in some cases, I wonder if it's truly impossible to fix in the others?

Also, it looks like `:not()` is supported all the way back to IE 9, so you could also try that for the exclusions: http://www.w3schools.com/cssref/sel_not.asp